tests/Makefile.am

```diff
@@ -12,7 +12,8 @@ EXTRA_DIST += local.at $(TESTSUITE)
 
 AM_CPPFLAGS = -I$(top_srcdir)/include
 
-rpmpgpcheck_LDADD = ../rpmio/librpmio.la
+rpmpgpcheck_LDADD = ../rpmio/librpmio.la \
+	@WITH_RPM_SEQUOIA_LIB@
```

Comment:
If I remove that and run `make check`, I get:

```
libtool: link: cc -Wall -Wpointer-arith -Wmissing-prototypes -Wstrict-prototypes -fno-strict-aliasing -fno-strict-overflow -fno-delete-null-pointer-checks -Wempty-body -g -O2 -o .libs/rpmpgpcheck rpmpgpcheck.o  ../rpmio/.libs/librpmio.so -ldl -lpthread -Wl,-rpath -Wl,/home/us/neal/work/pep/rpm/b/../i/lib
/usr/bin/ld: rpmpgpcheck.o: undefined reference to symbol 'pgpPrtParams'
/usr/bin/ld: /tmp/rpm-sequoia/debug/librpm_sequoia.so: error adding symbols: DSO missing from command line
clang: error: linker command failed with exit code 1 (use -v to see invocation)
make[3]: *** [Makefile:564: rpmpgpcheck] Error 1
``

Reply:
Doh, of course... because unlike with openssl/gcrypt, rpm-sequoia is what implements those functions when used.
This is not unlike static linkage which leaks the internal linkage to API users too, which is quite a PITA for the users.
At the very least @WITH_RPM_SEQUOIA_LIB@ needs to be added to rpm.pc.in so it gets pulled in through that, but I'd prefer hiding this all inside librpmio, which would require some kind of wrapping rather than the direct approach used now. But that's probably out of scope here.

Reply:
Ok, given that this is out of scope for this PR, I think it makes sense to open a separate issue once this PR is merged.

Reply:
I'd like to see rpm.pc.in updated in this PR though.

Reply:
I added `@WITH_RPM_SEQUOIA_LIB@` to `rpm.pc.in` and removed the explicit linkage in `tests/Makefile.am`, but then the test fails to link.  I think this is because the test doesn't actually use `rpm.pc`.  Should I leave the explicit linkage in `tests/Makefile.am` in addition to the rpm.pc.in change?

Reply:
Great!  Then the latest version of the PR does the right thing.